apps/sms/test/unit/information_test.js

```diff
@@ -324,7 +318,6 @@ suite('Information view', function() {
       }
 
       sinon.assert.called(reportView.renderContactList);
-      sinon.assert.calledWithMatch(ThreadUI.setHeaderContent, 'message-report');
```

Comment:
@steveck-chung I deleted this statement as we added l10n string to markup.Thank you.